ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -941,6 +941,77 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-advanced
+- as: aws-rosa-sts-advanced-f360-content
+  cron: 9 21 2,9,16,25 * *
+  steps:
```

Comment:
Wiring this up to notifications would be useful, too. We have a template we use in some of the other periodic jobs.

https://github.com/openshift/release/blob/master/ci-operator/jobs/ComplianceAsCode/content/ComplianceAsCode-content-master-periodics.yaml#L19-L29

Or are there notifications wired up somewhere else?